File: sort/buble_sort.py
arr = [21, 75, 40, 89, 72, 2, 45, 32, 28, 7, 56, 58, 60, 64, 50]

def buble_sort(a):
    for i in range(len(a), 0, -1):
        for j in range(i-1):
            if arr[j] > arr[j+1]:
                arr[j], arr[j+1] = arr[j+1], arr[j]

if __name__ == '__main__':
    buble_sort(arr)
    print(arr)

File: sort_js/buble_sort.js
var utils = require('./utils');

var arr = [21, 75, 40, 89, 72, 2, 45, 32, 28, 7, 56, 58, 60, 64, 50];


function buble_sort(a) {
    for (var i = a.length - 1; i > 0; i--) {
        for (var j = 0; j < i; j++) {
            if (a[j] > a[j + 1]) {
                utils.swap(a, j, j + 1);
            }
        }
    }
}

(function main() {
    buble_sort(arr);
    console.log(arr);
})();